src/requester/FakeRequester.js

```diff
@@ -8,11 +8,11 @@ export default class FakeRequester extends Requester {
   }
 
   addResponseFor({endpoint, response}) {
-    this._expectedResponses[endpoint.constructor.name] = response;
+    this._expectedResponses[endpoint.name] = response;
   }
 
   _getResponseClassFor(endpoint) {
-    return this._expectedResponses[endpoint.constructor.name];
+    return this._expectedResponses[endpoint.name];
```

Comment:
Si JS no permite redefinir el `==` una opción es definir un método `equals`, pero bueno no podrias usar aca `[]` tendrias que hacer algo de la pinta `Object.keys(this._expectedResponses).find( eachEndpoint) => eachEndpoint.equals(endpoint))`